ui/app/store/actions.js

```diff
@@ -2591,6 +2603,24 @@ function setUseBlockie (val) {
   }
 }
 
+function setUseNonceField (val) {
+  return (dispatch) => {
+    dispatch(actions.showLoadingIndication())
+    log.debug(`background.setUseNonceField`)
+    background.setUseNonceField(val, (err) => {
+      dispatch(actions.hideLoadingIndication())
+      !val && dispatch(actions.updateCustomNonce(''))
```

Comment:
I'm guessing this line was used to coerce `undefined`/`null` back to a string? Casting to a string in the handler would mean this line isn't needed anymore I suspect, as it'd always be a string.